### PR TITLE
Aries 2024 Q2 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by petermetz — APPROVED** [review]
LGTM

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+[Hyperledger Aries]: https://www.hyperledger.org/projects/aries
+[Aries Annual Report]: https://toc.hyperledger.org/project-reports/2024/2024-annual-Hyperledger-Aries.html
+[Credo-TS]: https://github.com/openwallet-foundation/credo-ts
+[Bifold Wallet]: https://github.com/openwallet-foundation/bifold-wallet
+[Aries Cloud Agent Python]: https://github.com/hyperledger/aries-cloudagent-python
+[Aries VCX]: https://github.com/hyperledger/aries-vcx
+[ABSA]: absa.co.za
+[Aries Framework Go]: https://github.com/hyperledger/aries-framework-go
+[Gen Digital]: https://www.gendigital.com
+[BBS Signatures Go]: aries-bbs-go
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]:
```suggestion
[BBS Signatures Go]: https://github.com/hyperledger/aries-bbs-go
```

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+* Load testing on ACA-Py as a server component produced good results. BC Gov's
+  tests showed that we had more than enough issuing capacity for our needs for
+  the forseeable future. The [Aries Akrida] tool developed and released by
+  [Indicio PBC] in response to a BC Gov "Code With Us" [Aries Load Test Generator]
+  open source procurement has proven to be an extremely capable Aries/DIDComm
+  load testing tool. Easy to use and very powerful!
+* The PR for removing the [Indy SDK] from ACA-Py is in review and will soon
+  be merged, eliminating the older component in favour of newer and more capable
+  replacements ([Aries Askar], [Indy VDR], and [AnonCreds-RS]). Good to see that
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]: Don't see a link for Indy VDR.

Comment (tkuhrt) [reply project-reports/2024/2024-Q2-Hyperledger-Aries.md]: Similarly, for AnonCreds-RS.

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  transition!
+  * **NOTE: THIS DOES NOT REMOVE INDY SUPPORT FROM ACA-PY!**
+* Support for AnonCreds in W3C VCDM Format in Aries has been implemented in
+  [Credo-TS] and [Bifold Wallet], with the PR to add support in [ACA-Py]
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]: Don't see a link for ACA-Py

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  expected to be merged soon.
+* Work has begun on adding [DIDComm v2] support in [ACA-Py].
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]: Don't see a link for DIDComm v2

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+[Indicio PBC]: https://indicio.tech
+[Aries Akrida]: https://github.com/Indicio-tech/aries-akrida
+[Aries Load Test Generator]: https://marketplace.digital.gov.bc.ca/opportunities/code-with-us/51d7c289-51a8-4307-939c-5a4271c7b2b6
+
+## Aries Framework Reports
+
+### [Aries VCX]
+
+Work continued on Aries VCX through the end of the quarter, and through the end of April,
+but has no longer funded. To be determined if someone else picks up the work, or if
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]:
```suggestion
but is no longer funded. To be determined if someone else picks up the work, or if
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Aries.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+[Unqualified DIDs to use `did:peer` DIDs]: https://github.com/hyperledger/aries-rfcs/pull/793
+[AnonCreds in W3C Verifiable Credentials Data Model Standard]: https://marketplace.digital.gov.bc.ca/opportunities/code-with-us/7afcbd7c-2bbc-41ed-bf27-b6ba6e2903c5
+
+# Maintainer Diversity
+
+There are currently 62 people on the 48 Hyperledger Aries teams representing at least 19 organizations.
+
+# Contributor Diversity
+
+In 2023Q3 there were contributions from 64 individuals representing at least 18 organizations.
```
Comment (denyeart) [inline project-reports/2024/2024-Q2-Hyperledger-Aries.md]: 2023Q3?

**Comment by swcurran** [comment]
Problems in Markdown and typos corrected -- thanks for checking, sorry for not using lint.

**Comment by non-fungible-nelson** [comment]
LGTM, great job on diversity of contributors

**Review by non-fungible-nelson — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Pull request merged**